stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/MessageListItemViewHolderFactory.kt

```diff
@@ -29,16 +30,21 @@ import io.getstream.chat.android.ui.message.list.adapter.viewholder.internal.Pla
 import io.getstream.chat.android.ui.message.list.adapter.viewholder.internal.ThreadSeparatorViewHolder
 
 public open class MessageListItemViewHolderFactory {
-
     internal lateinit var decoratorProvider: DecoratorProvider
 
     protected lateinit var listenerContainer: MessageListListenerContainer
         private set
 
+    private lateinit var attachmentViewFactory: AttachmentViewFactory
```

Comment:
Shouldn't this be `protected` like `listenerContainer`, so that anyone extending `MessageListItemViewHolderFactory` can have access to it to generate Views for attachments?

Reply:
Will fix that